libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/helpers/AuditExtensionsHelper.java

```diff
@@ -60,18 +62,29 @@ public static AuditRestrictedVc getRestrictedDataForAuditEvent(SignedJWT verifia
             return new AuditRestrictedVc(name, docExpiryDate);
         }
 
-        var brp = (JSONArray) credentialSubject.get(VC_BRP);
+        var brp = (JSONArray) credentialSubject.get(VC_RESIDENCE_PERMIT);
         if (brp != null && !brp.isEmpty()) {
             var docExpiryDate = ((JSONObject) brp.get(0)).getAsString(VC_EXPIRY_DATE);
             return new AuditRestrictedVc(name, docExpiryDate);
         }
 
-        var idCard = (JSONArray) credentialSubject.get(VC_NATIONAL_ID);
+        var idCard = (JSONArray) credentialSubject.get(VC_ID_CARD);
         if (idCard != null && !idCard.isEmpty()) {
             var docExpiryDate = ((JSONObject) idCard.get(0)).getAsString(VC_EXPIRY_DATE);
             return new AuditRestrictedVc(name, docExpiryDate);
         }
 
         return new AuditRestrictedVc(name);
     }
+
+    public static AuditRestrictedInheritedIdentity getAuditRestrictedInheritedIdentity(
```

Comment:
Can we make this name consistent with the ones above?

I don't mind too much which way we align them, but e.g. `getRestrictedAuditDataForF2F` (if that's what it's for) and `getRestrictedAuditDataForInheritedIdentity`?